email/debian/control

```diff
@@ -3,14 +3,14 @@ Section: mail
 Priority: extra
 Maintainer: James Carter <jcarter@bytemark.co.uk>
 Uploaders: Patrick J Cherry <patrick@bytemark.co.uk>, Steve Kemp <steve@bytemark.co.uk>
-Build-Depends: debhelper (>= 7.0.0), txt2man, gem2deb, dh-systemd
+Build-Depends: debhelper (>= 7.0.0), txt2man, gem2deb, dh-systemd, symbiosis-common
```

Comment:
This is so that any binaries with manpage generation can be run, as the often include the hack of doom to re-read the script, and output the first long comment, minus the schebang, which has bene written in a manpage-stylee.